pkg/receive/writer_test.go

```diff
@@ -174,3 +175,80 @@ func TestWriter(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkWriterTimeSeriesWithSingleLabel_10(b *testing.B)   { benchmarkWriter(b, 1, 10) }
```

Comment:
Do you mind adding comment with what options you recommend running this test e.g what `benchtime`? (:

Reply:
I never used non-default benchtime, are there reasons to choose non-default value here? Do you know a book that explain such things by any chance? :upside_down_face:

Reply:
Yes , it explains (: somrething in ch7, something in ch8